### ISSUE TITLE
Migration from Google Plus to Wallabag

### ISSUE THREAD

**Issue by TofPlay** [wallabag/wallabag#3740]
Hi,
With [Google Takeout](https://takeout.google.com/settings/takeout) I can export all my posts in json format 
![image](https://user-images.githubusercontent.com/1082222/46809349-8ec26000-cd6e-11e8-8bc8-16562420e0b9.png)
How can I import these posts into Wallabag?

**Comment by childnode** [comment]
mhh ... wallabag isn't a social network ;) but it might be interesting to get things imported using wallabag as an archive, but to make this useful aside the core features "read it later" and "clutter-free / relaxed reading" and "getting things read"

**Comment by TofPlay** [comment]
> wallabag isn't a social network ;)
Yes I know and I did not use Google+ for the social network
It was more to keep in one place the interesting articles that I found during the technology watch that I do daily. The fact that they are shared and just an edge effect linked to Google Plus's social network function.

> Perhaps you find a better product that fits your need? Don't forget: Share with us ;)
Wallabag is what I found closest to my need. If I can recover my Google+ posts, it would be really useful for me.
